components/tabs/tabs.js

```diff
@@ -0,0 +1,388 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../../helpers/queueMicrotask.js';
+import './tab.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { getNextFocusable, getPreviousFocusable } from '../../helpers/focus.js';
+import { ArrowKeysMixin } from '../../mixins/arrow-keys-mixin.js';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { findComposedAncestor } from '../../helpers/dom.js';
+import { LocalizeStaticMixin } from '../../mixins/localize-static-mixin.js';
+import {repeat} from 'lit-html/directives/repeat';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+//const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
+
+//const scrollButtonWidth = 56;
+
+class Tabs extends LocalizeStaticMixin(ArrowKeysMixin(RtlMixin(LitElement))) {
+
+	static get properties() {
+		return {
+			maxToShow: { type: Number },
+			_tabInfos: { type: Array }
+		};
+	}
+
+	static get styles() {
+		return [bodyCompactStyles, css`
+			:host {
```

Comment:
All these styles are the same as the original version, with the exception of updates for `:host(dir="rtl")`.